src/Components/Scenes/SideNavigation/RequestListItem/RequestListItem.js

```diff
@@ -85,7 +86,9 @@ const RequestListItem = ({
       </ListItem>
       <Collapse in={openIconThree} timeout="auto" unmountOnExit>
         <List component="div" disablePadding>
-          {requestListItemText}
+          {currentRole.governance || currentRole.approver === true
```

Comment:
I'd leave the `=== true` off - it works fine without that because it's just a boolean

Or, if we want to be that specific, for consistency it should also be on the first condition before the `||`

Reply:
Good catch I removed it as you said it's not necessary. Good catch 🙈